### DISCUSSION TITLE
Octopath Traveler

### DISCUSSION THREAD

**Discussion by patokki** [skyline-emu/skyline#475] [Game Requests]
Like super Mario odessey this is using unreal engine 4, I hope it will work as soon as possible

**Comment by sauradipdebnath** [comment]
It is hd-2d it means it's a 3d with 2d sprites

**Comment by PixelyIon** [comment]
Super Mario Odyssey does not use Unreal Engine 4 and I'm not sure where you read that. It uses an in-house engine made by Nintendo. Unreal Engine is full of fairly complex stuff, I do want Octopath Traveler to work eventually but it's going to take a while.
